CentralUnit/Communication/XbeeNetwork.cs

```diff
@@ -346,22 +357,33 @@ private void ProcessXbeePacket(XBeePacket packet)
         {
             if (packet != null)
             {
+                if (Log.IsDebugEnabled)
+                {
+                    Log.Debug(packet.ToPrettyString());
```

Comment:
I dont know a lot about this logging library. But i would assume Log.Debug only does anything if debug is enabled right?

Reply:
The logging method only logs when debug is enabled, but that means a string would be generated every time this code is called. 

Adding the if statement saves generating a string.